py/client/pydeephaven/table.py

```diff
@@ -97,6 +99,638 @@ def to_arrow(self) -> pa.Table:
         """
         return self.session.flight_service.do_get_table(self)
 
+    def drop_columns(self, cols: Union[str, List[str]]) -> Table:
```

Comment:
Do we need this equivalent copy in `Query`?

Reply:
Yes, it is there with a different return type (Query vs. Table) and the description is also different.